src/policy/policy.cpp

```diff
@@ -46,6 +46,8 @@ bool IsStandard(const CScript& scriptPubKey, txnouttype& whichType, const bool w
             return false;
         if (m < 1 || m > n)
             return false;
+    } else if (whichType == TX_HTLC) {
+        return false;
```

Comment:
@arielgabizon Would this actually reject all TX_HTLC items?  I had the impression or understand that there was a design such that these would actually be completed in most instances.  In lnd, a description of this was given as follows:  "If payment/htlc amount is too small, than such output is called dust. If for some reason channel have been force closed during payment, and dust htlc/payment haven't been settled (both sides haven't agreed to remove the htlc and change the balances accordingly), th(e)n we (do) not include it in commitment transaction (thereby giv(ing it) as a fee to miners) in order to make it valid."  For reference to this comment (from a contributor in lnd) see:  https://github.com/lightningnetwork/lnd/pull/115#issuecomment-305739226

With that said, I'm not sure of how the differences will play out between how bitcoin core is planning on handling it vs. how it is handled presently in the lnd development process.  

cc @ebfull